sdk/dotnet/Pulumi.Tests/Mocks/MocksTests.cs

```diff
@@ -81,16 +81,5 @@ public static Task<ImmutableArray<Resource>> RunAsync<T>() where T : Stack, new(
         {
             return Deployment.TestAsync<T>(new MyMocks(), new TestOptions { IsPreview = false });
         }
-
-        public static Task<T> GetValueAsync<T>(this Output<T> output)
```

Comment:
I expect the above tests to fail then? Or which extension method will they use instead?

Reply:
This particular method was unused as the above tests are directly calling the internal [Output\<T\>.GetValueAsync](https://github.com/pulumi/pulumi/blob/2e25e28debe7b798f25036ade8e9b250c30c5828/sdk/dotnet/Pulumi/Core/Output.cs#L150-L154).